### PR TITLE
Addresses issue #36 by adding license header checks. Added headers on…

### PR DESCRIPTION
**Pull request by zznate**
… missing files, added exlsion list for files that dont necessarily need them

### CODE REVIEW

File: jvector-base/pom.xml [hunk 1]
```diff
@@ -11,4 +11,15 @@
     </parent>
     <artifactId>jvector-base</artifactId>
```
Comment (jbellis) [inline jvector-base/pom.xml:12]: does this need to go in Base?

Comment (zznate) [reply jvector-base/pom.xml:12]: It's sort of annoying, but by design, Maven modules have all their variables scoped, so there is not a real easy way to refer to the configuration settings in the parent.  Placed here, it gets included as a transitive dependency by all the other modules.

Comment (jbellis) [reply jvector-base/pom.xml:12]: okay, and this is a "plugin" which is distinct from a "dependency" so jvector users aren't going to pull down rat as well, is that right?

Comment (zznate) [reply jvector-base/pom.xml:12]: Correct. The plugin machinery is specific to Maven workflows. It's stored in the local repository, but has nothing to do with the produced artifacts.

Comment (jbellis) [reply jvector-base/pom.xml:12]: Ship it!

**Pull request merged**